JavaScript/Topics/3.Web_Browser/README.md

```diff
@@ -1 +1,176 @@
-# To Do
\ No newline at end of file
+# JavaScript's usage in the Web Browser
+
+JavaScript is commonly utilised in both front-end and back-end web development. However, the focus of this lesson will be on front-end JavaScript implementation.
+
+## Where Javascript is used in Front-End?
+
+The structure of a web page is provided by HTML, the layout and rudimentary animations are provided by CSS, and the website's interactivity is provided by JavaScript.
+
```

Comment:
```suggestion

So, In a nutshell we can say :- HTML = NOUN, CSS = ADJECTIVE, JS = VERB

```

Please add this line, if you find it correct

Reply:
Oh, this looks short and precise! I will add it.

Reply:
if you solve the solns mentioned previous comments, then there is a button saying "RESOLVE" this, minimize these comments by clicking it